asmcli/lib/util.sh

```diff
@@ -565,3 +565,12 @@ EOF
     readonly GKE_CLUSTER_URI; readonly GCE_NETWORK_NAME;
   fi
 }
+
+# Reads in stdin, expected in the form of kubectl config get-contexts
+# output and isolates the default context's cluster value.
+get_context_cluster() {
+  local CONTEXT_ROW
+  while IFS= read -r CONTEXT_ROW; do
+    echo "${CONTEXT_ROW}" | grep '^\*' | cut -c2- | sed 's/\([[:blank:]]\)*\1/\1/g' | cut -d ' ' -f 3
+  done
+}
```

Comment:
I feel like you can just make the entire thing one line

```
get_context_cluster() {
  grep '^\*' | cut -c2- | sed 's/\([[:blank:]]\)*\1/\1/g' | cut -d ' ' -f 3
}
```

Reply:
yes, done.